homeassistant/components/ecobee/climate.py

```diff
@@ -628,6 +628,25 @@ def preset_modes(self):
         """Return available preset modes."""
         return list(self._preset_modes.values())
 
+    def set_fan_mode(self, fan_mode):
```

Comment:
Please keep functions in the same place in the file unless there is a good reason to move them as it makes it harder to review changes

Reply:
And even if there is a good reason for moving code around, that should be done in a separate PR which only does the restructuring but no functional changes IMO.